conda/conda-osx-64.lock.yml

```diff
@@ -374,8 +376,8 @@ dependencies:
   - jupyterlab=3.4.2=pyhd8ed1ab_0
   - ipyleaflet=0.16.0=pyhd8ed1ab_4
   - jupyter-sphinx=0.3.2=pyhd8ed1ab_1
-  - jupyterhub-singleuser=2.3.0=pyhd8ed1ab_0
+  - jupyterhub-singleuser=2.3.1=pyhd8ed1ab_0
   - myst-nb=0.13.2=pyhd8ed1ab_0
   - jupyter-book=0.12.3=pyhd8ed1ab_0
   - pip:
-    - jinja-markdown === 1.210911 
+    - jinja-markdown === 1.210911 --hash=md5:None
```

Comment:
This `--hash` change currently breaks the build.
I am looking into this on why this was added by the script

Reply:
Looks like [this step](https://github.com/snowex-hackweek/website2022/blob/d49ed4d8d8fc3f6bb82a8fba6408318ac9e2c58c/conda/lock-environment.sh#L35) in the `lock-environment.sh` file did succeed for you locally.

I will manually remove this hash which should fix this build

Reply:
I assume you are on a Mac?

Reply:
Yup - that explains it!